src/main/kotlin/com/cognifide/gradle/aem/common/file/resolver/Resolver.kt

```diff
@@ -92,24 +93,34 @@ val downloadDir: File
     }
 
     /**
-     * Resolve file in case of various type of specified value: file, url to file, dependency notation.
+     * Resolve file in case of various type of specified value: file, url to file, dependency notation, project dependency.
      */
-    fun get(value: Any): FileResolution = try {
-        useLocal(project.file(value))
-    } catch (e: InvalidUserDataException) {
-        if (value !is String) {
-            throw FileException("Cannot resolve file using value: $value")
-        }
-
-        aem.fileTransfer.handling(value).run {
-            resolveFileUrl(value, parallelable) { download(value, it) }
+    @Suppress("NestedBlockDepth")
```

Comment:
clue of change. now there is no unnecessary copying of file when file is being resolved from repository handled by Gradle dependency management

also support for external project dependencies is improved